### PR TITLE
Changed image iteration order to go through the reference image first.

### PR DESCRIPTION
**Pull request by kroczi**

### CODE REVIEW

File: gi_core/cninr_solver.cpp [hunk 1]
```diff
@@ -60,33 +60,33 @@ CninrSolver2D::CninrSolver2D(Image2D& reference, Image2D& target, double percent
...
-    for (int ty = 0; ty < tarShape[1]; ty++)
+    // Iterating over the reference image
+    for (int ry = 0; ry < refRShape[1]; ry++)
     {
-        for (int tx = 0; tx < tarShape[0]; tx++)
+        for (int rx = 0; rx < refRShape[0]; rx++)
         {
-            // Iterating over the reference image
-            for (int ry = 0; ry < refShape[1]; ry++)
+            /// Iterating over the target image
```
Comment (patrykks) [inline gi_core/cninr_solver.cpp]: Triple slash

File: gi_core/cninr_solver.cpp [hunk 2]
```diff
@@ -96,172 +96,172 @@ CninrSolver3D::CninrSolver3D(Image3D& reference, Image3D& target, double percent
...
+    for (int rz = 0; rz < refRShape[2]; rz++)
     {
-        int gz = tarShape[2] - 1 - tz;
-        for (int ty = 0; ty < tarShape[1]; ty++)
+        int gz = refRShape[2] - 1 - rz;
+        for (int ry = 0; ry < refRShape[1]; ry++)
         {
-            for (int rz = tarShape[2] - 1; rz >= 0; rz--)
+            /// Iterating over the target image
+            for (int tz = refRShape[2] - 1; tz >= 0; tz--)
```
Comment (patrykks) [inline gi_core/cninr_solver.cpp]: is it correct ??

Comment (kroczi) [inline gi_core/cninr_solver.cpp]: Correct? Surely. But it looks a bit pointless to invert the iteration order. You need to ask the author for his motives.

File: gi_core/cninr_solver.cpp (hunk 1 above)
Comment (kroczi) [inline gi_core/cninr_solver.cpp]: Done

**Pull request merged**